### PR TITLE
Feature/add abstract

### PR DESCRIPTION
**Pull request by y0z**
## Contributor Agreements

Please read the [contributor agreements](https://github.com/optuna/optunahub-registry/blob/main/CONTRIBUTING.md#contributor-agreements) and if you agree, please click the checkbox below.

- [x] I agree to the contributor agreements.

> [!TIP]
> Please follow the [Quick TODO list](https://github.com/optuna/optunahub-registry/tree/main?tab=readme-ov-file#quick-todo-list-towards-contribution) to smoothly merge your PR.

## Motivation &  Description of the changes

- Add Abstract section to README.md (First part of https://github.com/optuna/optunahub-registry/pull/129)

### CODE REVIEW

File: recipes/002_registration.py [hunk 1]
```diff
@@ -78,21 +78,23 @@
...
   - `license` (string): The license of the package. It should be a string. For example, `MIT License`. The license must be `MIT License` in the current version of OptunaHub.
 
-- `Class or Function Names` section that describes the classes or functions provided by the package. If you provide multiple classes or functions, you should list them in this section. Note that the section must be a markdown list. If you provide only one class or function, you can simply write the class or function name. Note that the documentation of the classes or functions must be written in their docstrings. If you want to refer to the documentation, please leave the source code link, or write them in the following `Others` section. For example:
+- `Abstract <https://github.com/optuna/optunahub-registry/blob/main/template/README.md#abstract>`__ section that describes the summary of your package. It should be a markdown paragraph. This section is helpful to advertise your package to potential users.
```
Comment (toshihikoyanase) [inline recipes/002_registration.py:81]: Not a strong opinion, but I may want to replace `advertise` with `attract`.
```suggestion
- `Abstract <https://github.com/optuna/optunahub-registry/blob/main/template/README.md#abstract>`__ section that describes the summary of your package. It should be a markdown paragraph. This section will help attract potential users to your package.
```

File: template/README.md [hunk 2]
```diff
@@ -31,6 +31,15 @@ Please read the [tutorial guide](https://optuna.github.io/optunahub-registry/rec
 You can find more detailed explanation of the following contents in the tutorial.
 Looking at [other packages' implementations](https://github.com/optuna/optunahub-registry/tree/main/package) will also help you.
 
+## Abstract
+
+You can provide an abstract for your package here.
+This section is helpful to advertise your package to potential users.
```
Comment (toshihikoyanase) [inline template/README.md:37]: ditto.
```suggestion
This section will help attract potential users to your package.
```

**Review by toshihikoyanase — COMMENTED** [review]
Thanks for your PR. I have some minor comments.

**Comment by y0z** [comment]
@toshihikoyanase 
Thank you for your feedback.
I applied your comments. PTAL.

**Review by toshihikoyanase — APPROVED** [review]
Thanks. Looks good to me.

**Pull request merged**